setup.py

```diff
@@ -43,28 +44,50 @@ def get_extensions():
 
     main_source = path.join(extensions_dir, "vision.cpp")
     sources = glob.glob(path.join(extensions_dir, "**", "*.cpp"))
-    source_cuda = glob.glob(path.join(extensions_dir, "**", "*.cu")) + glob.glob(
-        path.join(extensions_dir, "*.cu")
-    )
+
+    is_rocm_pytorch = False
+    if torch.__version__ >= '1.5':
```

Comment:
not sure if it will happen, but '1.10' < '1.5' in terms of string. You can compare by tuple or list (like line 14)

Reply:
Agreed.